### DISCUSSION TITLE
[Unknown Mod][Spells] Tweaks to Hypnotize spell

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2517] [Ideas]
As you may have noticed, it is extremely rare to use the Hypnotize spell. :)

I would like to make the Hypnotize spell works like the others Mind spells (Blind, Paralyze and Berserk), the hero can cast the spells on an enemy stack no matter the number of creatures in this stack.

The Hypnotize spell will continue to work the exact same way:
- the duration of the spell is egal to the hero's Spell Power (hero's Spell Power 3, spell duration 3 turns),
- the spell can be canceled by the Dispel Magic spells, the Cure spells and the Anti-Magic spell,
- the spell is canceled if the hypnotized stack performs an attack,
- the spell is not canceled when the hypnotized stack is physically attacked (melee attack, ranged attack), and the hypnotized stack doesn't retaliate,
- the spell is not canceled when the hypnotized stack takes damage from damage spells.

So, just change the way the hero is allowed to cast the spell, on an "entire stack" instead of "if the hit points of the stack are less than 25x Power".

**Edit 2023/04/19.**

In the original game, the Berserker and Hypnotize spells cancel each other out.
In fact it doesn't really matter in the original game, since the use of the Hypnotize spell is really not interesting due to its limitations, and this spell is used very rarely or not at all. :)

With the change in the way the Hypnotize spell works in the mod, the spell becomes more interesting.
So it will be possible to cast the Berserker and Hypnotize spells simultaneously on the same troop.
The Hypnotize spell will have priority over the Berserker spell, as it is already the case for the Blind and Paralyze spells.
This means that the Berserker spell will only take effect when the Hypnotize spell is no longer active.

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu !
Don't you think you make this spell too powerful? :D
I can agree with you, that spell requires reformation... But not sure so drastic...)
Did you think about more minor improvements? Like updating the current formula but like 2x/3x/4x rate? 
Or adding additional rule for your post above, which allows to control the target only the next turn...?

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> Hi, @LeHerosInconnu !
> Don't you think you make this spell too powerful? :D
> I can agree with you, that spell requires reformation... But not sure so drastic...)
> Did you think about more minor improvements? Like updating the current formula but like 2x/3x/4x rate?
> Or adding additional rule for your post above, which allows to control the target only the next turn...?

I don't think it will make it too powerful.

Lots of the creatures can not be affected by the spell.
Green, Red and Black Dragons are immune to all spells; Giant, Titan, all Necromancer creatures, Ghost, and all Elementals are immune to Mind spells.

And it can be cancelled quite easily; there is 5 spells for this, Dispel Magic, Mass Dispel, Cure, Mass Cure and Anti-Magic; and there is always at least one of these spells in every Mage Guild.

The spell makes no damage by itself, you still need to damage the creatures with physical attacks or with damage spells.

And it's a level 5 spell, it should have a certain power, a certain value.
It can be seen as an improved version of the Berzerker spell (and also of the Blind and Paralyze spells), because it allows the player to choose the troop that will be attacked (or not to attack).

Modifying its functioning with 2 times, 3 times, etc. the ability to cast the spell will just postpone the original problem.
For my mod, I prefer a spell that works in all conditions, so that the player can finally use it (and not complain when it appears in a Mage Guild).

This may also be an option for players who wish to continue not to be able to use it, but I doubt there are many players in this case. :D

**Reply by Branikolog** [reply]
I don't know, how it was implemented in later series, but in HoMM5 it was quite balanced: if hero had not enough SP he couldn't succeed in hypnotizing especially strong creature... So he had to choose the target according to his SP.
Also, HoMM2 offers quite smaller armies, due to only 5 slots in garrison... So I'm not so enthusiastic about making this spell 100% guarantee on any creature without immune....
Offtopic: Could you please explain difference between paralyze and blind?

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> I don't know, how it was implemented in later series, but in HoMM5 it was quite balanced: if hero had not enough SP he couldn't succeed in hypnotizing especially strong creature... So he had to choose the target according to his SP.
> Also, HoMM2 offers quite smaller armies, due to only 5 slots in garrison...

A remake of Heroes 2 must remain simple and clear in its use.

Here, all the mind affecting spells would work the same way, no questions to ask, no calculations to make, take a decision and apply it.

> So I'm not so enthusiastic about making this spell 100% guarantee on any creature without immune....

The beauty of an option is that you don't have to use it.
This is part of my mod to make the game fun to play in single player mode. :)

> Offtopic: Could you please explain difference between paralyze and blind?

Under the influence of the Blind spell, the attacked troop retaliates with only half the damage.
Under the influence of the Paralyze spell, the attacked troop does not retaliate.

**Comment by lioror** [comment]
I agree heroes2 needs a balance mod.
I think hypnotize is closer to an ability the hero has, rather then spell.
So I changed it to have chance of success based on stack hp.
For example, if the hero can hypnotize 500 hp and you target a stack with 1000 hp, it has 50% chance to succussed.

The spell icon was not captured.
![image](https://github.com/ihhub/fheroes2/assets/6975903/a1385ebc-6d06-4c4e-bfb0-85e6484a48cb)

**Reply by LeHerosInconnu** [reply]
Hello @lioror,

Interesting way of doing things. :)

For me, the problem with spells with a percentage of success is that, in the long run, the user will always end up using another spell whose effects he's sure to be reliable.
The 15 spell points of the spell cost are still a fairly large number of spell points, and the player can use them with other spells that work for sure.

Also, in Heroes 2 during combat, all spells always work for sure, so there's no mechanism for casting a spell in the hope that it will work.
Even for a spell cast on dwarves, who have a percentage of resistance to magic, it's the dwarf's ability that makes the spell have no effect.
The player is aware of the dwarf's resistance to magic and decides whether or not to cast the spell on the dwarf.

So you're also working on a mod for fheroes2?

**Reply by lioror** [reply]
It's still up to you to decide which spell to use.
In some cases damaging spell is what you want, in other cases you might prefer a different tactic.
Controlling an enemy unit is very strong effect.

I am experimenting with things in fheroes2.
I want to do a balance mod, I think about starting a discussion about it.
I would like to get feedback and ideas.
